src/styles/_themes.scss

```diff
@@ -152,52 +152,52 @@ $themes: (
     colorError: $colorError,
 
     // App - START
-    colorSelectionText: #212121,
+    colorSelectionText: #333333,
```

Comment:
Whenever you have a color hex value in css that is a repetition of the same value, you can truncate it to a three character shorthand. e.g. `#333333` becomes `#333`. We always use the shorthand when possible ([see the code style guide](https://github.com/Azure/pcs-remote-monitoring-webui/wiki/Code-Guide#css)).